### PR TITLE
How to identify suspcious email

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by stephamon** [comment]
Hey @cyrichardson can you take a look at this new article please?

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-30bb9c259e/how-to/)

**Comment by cyrichardson** [comment]
Will do, @stephamon.

**Comment by cyrichardson** [comment]
@stephamon Suggested changes here: 
[identifying-suspicious-email_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2411640/identifying-suspicious-email_cyrEdits.docx)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-86238438b6/how-to/). :bow:

File: content/rackspace-email/identifying-suspicious-email.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+<img src="{% asset_path rackspace-email/identifying-suspicious-email/this-is-suspicious.png %}" />
+
+Second, Webmail clearly displays the friendly name and email address of the sender. Comparing the name to the email address is a simple way to check for display name spoofing. Display name spoofing is when bad guys put a name that you recognize in the **From** address, such as the name of your manager, but the associated email address is not for that person. For example, Webmail shows you "From: John Doe <sally@notmycompanydomain.com>". Assuming John Doe is your manager's name, sally@notmycompanydomain.com is definitely not your manager's email address. In this case, the bad guy is using a compromised mailbox on another system to pose as your manager.
```
Comment (RackerKate) [inline content/rackspace-email/identifying-suspicious-email.md]: Suggest putting the period at the end of the 4th sentence inside the parens.

File: content/rackspace-email/identifying-suspicious-email.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+<img src="{% asset_path rackspace-email/identifying-suspicious-email/sender-discrepancies.png %}" />
+
+Finally, Webmail tells you if the domain of the sender does not match the domain used to send the message. This is represented by the **'sent from'** added to the sender's address. In some cases, this is normal for sending services used in marketing and sales campaigns, newsletters, and so on. However, it can also be an indication of spoofing. Building on the above example, if the same bad guy has compromised a Gmail mailbox and is sending email pretending to be your manager, it shows up in Webmail like this: "From: John Doe <john.doe@mycompany.com> sent from gmail.com". Assuming your manager never uses Gmail for work purposes, this is a clear indicator that the message is suspicious.
```
Comment (RackerKate) [inline content/rackspace-email/identifying-suspicious-email.md]: Suggest "The domain that sent the message is represented by..." 

Suggest "In some cases, this situation is normal for sending services that are used for marketing and sales campaigns, newsletters, and similar email distributions."

Suggest "Building on the previous example...it appears in Webmail as 'From: John Doe <john.doe@mycompany.com> sent from gmail.com.'"

File: content/rackspace-email/identifying-suspicious-email.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+<img src="{% asset_path rackspace-email/identifying-suspicious-email/sender-discrepancies-2.png %}" />
+
+The suspicious message banner, the full **From:** name and email address and the **sent from** domain, help you ensure sure the messages you interact with are from who you think they are from. Additionally, it is always a best to verify any request for personal information or money that is received through email. For more information on how to recognize phishing emails,  visit our blog here: https://blog.rackspace.com/email-phishing-rise-mailbox-safe.
```
Comment (RackerKate) [inline content/rackspace-email/identifying-suspicious-email.md]: "The suspicious message banner, the full **From:** name and email address, and the **sent from** domain help you ensure that the messages you interact with are from the senders that you expect. Additionally, it is always a best practice to...For more information on how to recognize phishing emails, see [Email Phishing on the Rise: Is Your Mailbox Safe?](https://blog.rackspace.com/email-phishing-rise-mailbox-safe).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b2f008b394/how-to/)

File: content/rackspace-email/identifying-suspicious-email.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+<img src="{% asset_path rackspace-email/identifying-suspicious-email/sender-discrepancies.png %}" />
+
+Finally, Webmail tells you if the domain of the sender does not match the domain used to send the message. The domain that sent the message is represented by the **'sent from'** added to the sender's address. In some cases, this situation is normal for sending services that are used for marketing and sales campaigns, newsletters, and similar email distributions. However, it can also be an indication of spoofing. Building on the previous example, if the same bad guy has compromised a Gmail mailbox and is sending email pretending to be your manager, it appears up in Webmail as "From: John Doe <john.doe@mycompany.com> sent from gmail.com." Assuming your manager never uses Gmail for work purposes, this is a clear indicator that the message is suspicious.
```
Comment (RackerKate) [inline content/rackspace-email/identifying-suspicious-email.md]: "...it appears in Webmail as..." That might have been my bad.

File: content/rackspace-email/identifying-suspicious-email.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+<img src="{% asset_path rackspace-email/identifying-suspicious-email/sender-discrepancies-2.png %}" />
+
+The suspicious message banner, the full **From:** name and email address and the **sent from** domain help you ensure sure that the messages you interact with are from the senders that you expect. Additionally, it is always a best practice to verify any request for personal information or money that is received through email. For more information on how to recognize phishing emails,  see https://blog.rackspace.com/email-phishing-rise-mailbox-safe.
```
Comment (RackerKate) [inline content/rackspace-email/identifying-suspicious-email.md]: Suggest removing the "sure" after "ensure."

Also remove extra space before "see" in the last line here.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-38f256d671/how-to/).

**Pull request merged**